test/test.ui.js

```diff
@@ -515,26 +546,13 @@ test('Geocoder#addTo', function(tt) {
     opts.accessToken = mapboxgl.accessToken;
     opts.enableEventLogging = false;
     container = document.createElement('div');
-    container.className = "notAMapEither"
     document.body.appendChild(container)
     geocoder = new MapboxGeocoder(opts);
-    geocoder.addTo(document.querySelectorAll(".notAMapEither"));
+    geocoder.addTo(container);
     t.ok(Object.keys(container.getElementsByClassName("mapboxgl-ctrl-geocoder--input")).length, 'geocoder exists when added to an html element')
     t.end(); 
   });
 
-  tt.test('throws if added to an unknown element', (t)=>{
```

Comment:
I agree, I'm not sure what this unit test is for. @scottsfarley93 added it in https://github.com/mapbox/mapbox-gl-geocoder/commit/7f53ecf331972161e12691e0fd9e94974d3ff46a#diff-23a1ea5628fe499863040c05a9d338aaR440 but as far as I can tell it's checking to make sure the element is a map, but in that PR at https://github.com/mapbox/mapbox-gl-geocoder/pull/295 support was added to allow adding the geocoder to a non-map element for a mapless geocoder.